views/movies/new-movie.hbs

```diff
@@ -0,0 +1,18 @@
+<h2>Create a new movie!</h2>
+
+<form action="/movies/create" method="POST">
+
+<label>Title
+    <input type="text" name="title"/>
+</label>
+<label>Genre
+    <input type="text" name="genre"/>
+</label>
+<label>Plot
+    <input type="text" name="plot"/>
+</label>
+<label>Cast
+    <input type="text" name="cast"/>
+</label>
```

Comment:
When creating new movie, havig input is not enough. Our movie model will only recognise celebrity id (from database), so as the lab instruction suggests we must have select + option tags, and send selected celebrity in the body, rather than typed name. For example:

```
<select name="cast" multiple>
    {{#each celebrities}}
           <option value="{{_id}}">{{name}}</option>
    {{/each}}
</select>
```